app/src/main/java/com/example/universitieslist/util/DispatcherHelper.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.universitieslist.util
+
+import kotlinx.coroutines.Dispatchers
+
+class DispatcherHelper() {
```

Comment:
Consider making this an interface, then, create a class to implement the interface. It will be useful in the test class i.e. just call the class directly in the VMTest class constructor.